components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/client/IdentityProviderMgtServiceClient.java

```diff
@@ -272,7 +281,7 @@ public void addIdP(IdentityProvider identityProvider) throws Exception {
             idPMgtStub.addIdP(identityProvider);
         } catch (Exception e) {
             log.error("Error in adding a Identity Provider for a given tenant", e);
-            throw new Exception(e.getMessage());
+            throw new Exception("Error in adding a Identity Provider for a given tenant", e);
```

Comment:
Is it possible to avoid changing the error message?

Reply:
Fixed